src/gui/addnewtorrentdialog.cpp

```diff
@@ -175,6 +178,35 @@ void AddNewTorrentDialog::setTopLevel(bool value)
     SettingsStorage::instance()->storeValue(KEY_TOPLEVEL, value);
 }
 
+int AddNewTorrentDialog::savePathHistoryLength()
+{
+    return savePathHistoryLengthSetting();
+}
+
+namespace
+{
+    const int minPathHistoryLength = 0;
+    const int maxPathHistoryLength = 99;
```

Comment:
Can you use this to setRange() for spinner?

Reply:
@Chocobo1 asked to remove that call because it duplicated default values. While there was no sense at all in the call with explicit numbers, it would bear some interest with names constants, I agree. @Chocobo1?

Reply:
OK.

Reply:
How can we be sure that it will not be changed in the future, and it will cease to match our value?